detections/endpoint/malicious_powershell_process___encoded_command.yml

```diff
@@ -56,3 +56,18 @@ tags:
   - Processes.dest
   - Processes.process_id
   security_domain: endpoint
+  impact: 70
+  confidence: 50
+  risk_score: 35
+  context:
+  - Source: Endpoint
```

Comment:
minor thing, all of these under context are meant to be simple string eg: `Source:Endpoint` no space in between.

Reply:
fixed these across all detections